lua/null-ls/builtins/diagnostics/cue_fmt.lua

```diff
@@ -0,0 +1,46 @@
+local h = require("null-ls.helpers")
+local methods = require("null-ls.methods")
+
+local DIAGNOSTICS = methods.internal.DIAGNOSTICS
+
+return h.make_builtin({
+    name = "cue_fmt",
+    method = DIAGNOSTICS,
+    filetypes = { "cue" },
+    generator_opts = {
+        command = "cue",
+        args = { "fmt", "$FILENAME" },
+        format = "raw",
+        from_stderr = true,
+        to_temp_file = true,
+        check_exit_code = function(code)
+            return code <= 1
+        end,
+        on_output = function(params)
```

Comment:
Your issue is most likely that setting `format = "raw"` requires returning diagnostics via the `done` callback, described [here](https://github.com/jose-elias-alvarez/null-ls.nvim/blob/main/doc/HELPERS.md#format). 

```suggestion
        on_output = function(params, done)
```

Reply:
That did it, thanks so much! I poured over the docs but somehow missed that section. 🤦‍♂️